agilent_infiniium_scope.py

```diff
@@ -366,18 +371,27 @@ def get_trace(self, c, channel, start=1, stop=10000):
         #Transfer waveform preamble
         preamble = yield dev.query('WAV:PRE?')
         #Transfer waveform data
-        binary = yield dev.query('WAV:DATA?')
+        p = dev._packet(). \
+            write('WAV:DATA?'). \
+            read_raw()
```

Comment:
styleguide says: no backslash line continuation (https://google.github.io/styleguide/pyguide.html#Line_length)

In this case, seems fine to just put on one line
```python
p = dev._packet().write('WAV:DATA?').read_raw()
```